hypervisor/arch/x86/configs/up2-sbl.config

```diff
@@ -0,0 +1,5 @@
+CONFIG_BOARD="SBL-UP2"
```

Comment:
What is the reason you are calling this `SBL-UP2` and not the other way around, i.e. `UP2-SBL`?